### ISSUE TITLE
Extreme Lag opening Godot 4.1.3 Stable, projects and running the project

### ISSUE THREAD

**Issue by sasharp65** [godotengine/godot#84806]
### Godot version

4.1.3 Stable

### System information

Godot v4.1.3.stable - Windows 10.0.19045 - Vulkan (Mobile) - dedicated NVIDIA GeForce RTX 3080 (NVIDIA; 31.0.15.3667) - Intel(R) Core(TM) i7-9700K CPU @ 3.60GHz (8 Threads) - 32 GB RAM

### Issue description

I just downloaded Godot 4.1.3 stable because I want to learn Game Dev in my retirement. 

There is extreme lag and delay when:

1. Opening Godot - 42 seconds
2. Opening Project - another 45 seconds
3. Play/Run the Project - another 41 seconds

I thought my Anti Virus might be interfering so I disabled that to test and still have the same issues.

Although 4 years old, my PC would still be considered and upper tier Gaming rig.
I do not have issues with any other program and can play any game on extreme resolutions (3x 2k monitors), even Diablo 4 maxed out.

### Steps to reproduce

Not much to write here, it happens every single time.
I just have one small project, one scene and the Godot Icon image added as a 2Dsprite.

### Minimal reproduction project

N/A

**Comment by sasharp65** [comment]
So I just happened to read someone else who said they had a long time to open project with Discord open.
I also had Discord open.

I closed Discord but still had the long wait after closing and reopening Godot.

5 - 10 minutes later Godot was opening up normal 

Godot Opening - < 1 sec
Project Opening  - 3 sec
Project Run - 1 sec

So I rebooted and wanted to see if Discord was still causing the issue.

After a clean reboot and Discord running, Godot was still running properly???

Note I did do a reboot a few times earlier with the same laggy results.

It was only after closing Discord once with Godot open that it eventually started working correctly.

Anyway...it seems to be working normally at the moment, so I will keep and eye on it.

**Comment by sasharp65** [comment]
Looks like I spoke too soon, the issues are back and Godot is in an unusable condition.
I'm getting the same 45 second delays after Opening Godot, opening a project and running a project.

After returning home and starting my PC up I tried opening Godot and getting laggy results previously mentioned.
I've closed Discord, Anti-Virus and most programs that I could from my task bar but I have the same lag.

I've been trying for 30 minutes but the same lag.
In each of those 3 phases Task Manager reports Not Responding for the Godot Engine for most of the time before it opens up.
![Godot_Not_Responding](https://github.com/godotengine/godot/assets/16639517/59735c16-3353-4b36-b18d-f2146d86a703)

**Comment by bitsawer** [comment]
This could be a possible duplicate of https://github.com/godotengine/godot/issues/84780 or https://github.com/godotengine/godot/issues/20566 or some combination of them.

You can try some of the workarounds discussed in the issues like unplugging any USB devices to see if it helps.

**Comment by sasharp65** [comment]
I started unplugging USB devices to test.  I do have a Focusrite AD converter and thought for sure that would be the problem after reading the two other tickets  you listed.  However, that wasn't the problem.

As soon as I unplugged my USB Corsair Strafe RGB mechanical keyboard Godot opened immediately.  Even after I plugged the keyboard back in it opens immediately.

If it matters, the keyboard uses two USB connections which I have connected to my monitor's USB built in hub which is needed because the cables wouldn't be long enough due to my standup desk.
